groupby post

quasiben opened the pull request:
This is rather rough but starts a post on the guts of groupby aggregations. @jrbourbeau feel free to push commits as you see fit

quasiben committed:
initial commit of groupby post

jrbourbeau commented:
Thanks for pinging me @quasiben! Just wanted to touch base and let you know that I'll look over this tomorrow

quasiben committed:
rework intro a bit

jrbourbeau committed:
Round of small edits

quasiben committed:
a few typo corrections and adding section on mean

quasiben committed:
resize images

quasiben committed:
size images and anchor them

quasiben committed:
add split apply combine

jrbourbeau committed:
Another round of minor edits

jrbourbeau commented:
Thanks for putting this all together @quasiben. I just pushed a few final minor edits. I'm happy with the post if you are : )

quasiben committed:
image tagline and change the date

quasiben commented:
@jrbourbeau thank you!  In the last commit I added the citation tag fo the split-apply-combine  image.  I also changed the date for today.  I'm happy with the post.  Should we merge now ?
